cybergis/general/comet.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+
+from .base import SBatchScript
+from .job import SlurmJob
+from .connection import SSHConnection
+
+
+logger = logging.getLogger("cybergis")
+
+
+class CometSBatchScript(SBatchScript):
+    pass
+
+
+class CometJob(SlurmJob):
+
+    JOB_ID_PREFIX = "Comet_"
+    backend = "Comet"
+    connection_class = SSHConnection
+    sbatch_script_class = CometSBatchScript
+
+    def prepare(self):
+        raise NotImplementedError()
+
+    def _save_remote_id(self, msg, *args, **kwargs):
```

Comment:
@CarnivalBug Same issue here? if you copied the entire function from base without any changes, then you can safely remove it here